aries-site/src/examples/components/button/DosDonts/ButtonGoodCancelPreview.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button, Heading, Header, Text } from 'grommet';
+
+export const ButtonGoodCancelPreview = () => (
+  <Box width="large" gap="medium" pad="small">
+    <Header
+      direction="column"
+      align="start"
+      gap="xxsmall"
+      pad={{ horizontal: 'xxsmall' }}
+    >
+      <Heading level={2} margin="none">
+        Delete
+      </Heading>
+      <Text size="xlarge">
```

Comment:
```suggestion
      <Text size="large">
```
Typically, I've seen this as size large.

Reply:
maybe I should change to `heading` in designs she had `27px`